src/Shared/Microsoft.DotNet.GitHub.Authentication/IGitHubAppTokenProvider.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.GitHub.Authentication
```

Comment:
Yeah I think this is the thing. This interface is introducing a new namespace instead of using the Dotnet with lowercase n that the classes are in.

Reply:
there are other files with the same prob -> 
![image](https://user-images.githubusercontent.com/47157394/79266733-e8688680-7e4c-11ea-8680-80452ed4c829.png)

but this is the folder structure 

![image](https://user-images.githubusercontent.com/47157394/79266761-f9b19300-7e4c-11ea-9c53-00e8c53e5ebd.png)

Reply:
I will change all of them to use Microsoft.DotNet.GitHub.Authentication in that folder.

Reply:
ah you mentioned the same file :)

Reply:
We're inconsistent in naming? say that ain't so! Thanks for taking a look haha.

Reply:
created an issue -> https://github.com/dotnet/core-eng/issues/9625, so resolving this comment.

Reply:
Thanks!